### PR TITLE
Add closure to multiqc ext.args

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

### CODE REVIEW

**Review by mahesh-panchal — APPROVED** [review]

**Review by ewels — APPROVED** [review]
LGTM, but 👉🏻  changelog :)

File: nf_core/pipeline-template/conf/modules.config [hunk 1]
```diff
@@ -15,14 +15,14 @@ process {
     publishDir = [
         path: { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" },
         mode: params.publish_dir_mode,
-        saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        pattern: "{!versions.yml}"
```
Comment (mahesh-panchal) [inline nf_core/pipeline-template/conf/modules.config:18]: I thought this was a shell pattern. Since when has this been possible?

Comment (mahesh-panchal) [reply nf_core/pipeline-template/conf/modules.config:18]: It's not working for me:
```nextflow
workflow {
  TASK()
}

process TASK {
    input:

    script:
    """
    touch versions.{yml,txt}
    """

    output:
    path "versions.yml"
    path "versions.txt"
    
}
```
```
process.publishDir = [ path: "results", pattern: "{!versions.yml}" ]
```
Nothing is output in this case, except the folder is made.

Comment (maxulysse) [reply nf_core/pipeline-template/conf/modules.config:18]: ok let me fix that then

Comment (maxulysse) [reply nf_core/pipeline-template/conf/modules.config:18]: yeah, you're right, not working, I should have check more my outputs

**Pull request merged**